yarn.lock

```diff
@@ -35810,21 +35810,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"ws@npm:^8.11.0, ws@npm:^8.13.0, ws@npm:^8.17.1, ws@npm:^8.2.3":
-  version: 8.18.0
```

Comment:
I know we are just making it resolve to a 1 minor version upgrade, but there is a remote possibility it might impact something else in Calypso. Why risk it? Or you relying on all tests being passing?

Reply:
The `dedupe` fails. https://github.com/Automattic/wp-calypso/blob/trunk/.teamcity/_self/projects/WebApp.kt#L437

Once followed [the documentation](https://github.com/Automattic/wp-calypso/blob/trunk/docs/dependency-management.md#list-duplicated-dependencies), this is what we get in the lock file. I don't know if there are any other implications whatsoever.

@alshakero, can you give us a hint here, when you have time? I appreciate any help you can provide.

Reply:
Yeah I think this is perfectly fine.

Reply:
Thank you!